pkg/aws/policies.go

```diff
@@ -1171,21 +1177,21 @@ func (c *awsClient) detachAttachedRolePolicies(role *string) error {
 }
 
 func (c *awsClient) DeleteInlineRolePolicies(role string) error {
-	listRolePolicyOutput, err := c.iamClient.ListRolePolicies(&iam.ListRolePoliciesInput{RoleName: aws.String(role)})
+	listRolePolicyOutput, err := c.iamClient.ListRolePolicies(context.Background(),
+		&iam.ListRolePoliciesInput{RoleName: aws.String(role)})
 	if err != nil {
 		return err
 	}
 	for _, policyName := range listRolePolicyOutput.PolicyNames {
-		_, err = c.iamClient.DeleteRolePolicy(&iam.DeleteRolePolicyInput{
-			PolicyName: policyName,
-			RoleName:   aws.String(role),
-		})
+		pn := policyName
+		_, err = c.iamClient.DeleteRolePolicy(context.Background(),
+			&iam.DeleteRolePolicyInput{
+				PolicyName: &pn,
```

Comment:
Would that work? or it won't be allowed to return the address of an entry?
```suggestion
				PolicyName: aws.string(policyName),
```

Reply:
no it requires a string pointer